.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea/
+.venv/
```

Comment:
@Serg-Bartosh , plz add `.some-hidden-dir/` as well

Reply:
@s373r , it was fixed

Reply:
@Serg-Bartosh , cool! :tada: +rep